templates/ep19/bs/user_panel/dashboard.html

```diff
@@ -49,12 +49,38 @@ <h2 style='margin-top: 2em'>Invoices</h2>
             {% endif %}
 
 
-        {% if proposals %}
-        {% include "ep19/bs/user_panel/_proposals.html" %}
-        {% endif %}
-
     </div>
     <div class="col-md-4">
+        <h2>Got questions?</h2>
+        You can ask them in our helpdesk
+        NOTE(artcz) Fix styling, this is temporary location
```

Comment:
This should probably be a comment

Reply:
this will need to be decided/removed before merge, that's why it visible. :)